examples/extensions/add_prices.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example adds a price extension and associates it with an account.
+
+Campaign targeting is also set using the specified campaign ID. To get
+campaigns, run basic_operations/get_campaigns.py
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.v2.services.campaign_service_client import (
+    CampaignServiceClient
+)
+
+
+def main(client, customer_id, campaign_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID str.
+        campaign_id: a campaign ID str.
+    """
+    # Create the price extension feed item
+    price_feed_item = client.get_type('PriceFeedItem', version='v2')
+    price_feed_item.type = (
+        client.get_type('PriceExtensionTypeEnum', version='v2').SERVICES)
+    # Optional: set price qualifier
+    price_feed_item.price_qualifier = (
+        client.get_type('PriceExtensionPriceQualifierEnum').FROM)
+    price_feed_item.tracking_url_template.value = (
+        'http://tracker.example.com/?u={lpurl}')
+    price_feed_item.language_code.value = 'en'
+
+    # To create a price extension, at least three price offerings are needed.
+    price_feed_item.price_offerings.extend([
+        create_price_offer(client,
+                    'Scrubs',
+                    'Body Scrub, Salt Scrub',
+                    60000000,  # 60 USD
+                    'USD',
+                    client.get_type('PriceExtensionPriceUnitEnum').PER_HOUR,
+                    'http://www.example.com/scrubs',
+                    'http://m.example.com/scrubs'),
+        create_price_offer(client,
+                    'Hair Cuts',
+                    'Once a month',
+                    75000000,  # 75 USD
+                    'USD',
+                    client.get_type('PriceExtensionPriceUnitEnum').PER_MONTH,
+                    'http://www.example.com/haircuts',
+                    'http://m.example.com/haircuts'),
+        create_price_offer(client, 'Skin Care Package',
```

Comment:
*Nitpick*
I don't think there's anything wrong with providing multiple arguments on a line, but it does break consistency when compared to the previous lines.

Reply:
Given the number of arguments, I think it is cleaner this way.

Reply:
Changed my mind after removing the extra get_types()